UI/window-basic-settings.hpp

```diff
@@ -170,6 +171,7 @@ class OBSBasicSettings : public QDialog {
 		outputsChanged = false;
 		audioChanged   = false;
 		videoChanged   = false;
+		screenshotsChanged	= false;
```

Comment:
Found another case of weird formatting, you probably want a space instead of a tab here